en/docs/learn/multi-channel-self-registration-and-account-confirmation.md

```diff
@@ -256,6 +249,43 @@ Management Policies** section.
         </tbody>
         </table>
         
+## Adding SMS notification templates
+
+The templates for the EMAIL notification are stored in `<IS_HOME</repository/conf/email 
+/email-admin-config.xml` file and the templates can be edited from the management console.
+
+!!! tip
+    The **AccountConfirmation** template is used to send email notifications.You can edit and 
+    customize the email template. For more information, 
+    see [Customizing Automated Emails](../../learn/customizing-automated-emails).
+    
+The templates for SMS notifications are stored in the registry. Follow the steps below to add
+SMS notification templates.
+
+!!! Warning
+    All the configs are case sensitive. Changing the case might break the functionality
+
+1. Login to the management console and navigate to `Main> Registry> Browse`.
+2. In **tree view tab**, click `system>config>identity`.
+3. Click **Add Collection** and enter collection name as `sms`.
+![add-sms-collection](../assets/img/learn/self-registration/adding-sms-templates/add-sms-collection.png)
+4. Click on the **sms directory** and click **Add Collection** and enter collection name as 
+`accountconfirmation`. This will create another directory inside the sms directory.
+![add-accountconfirmation-collection](../assets/img/learn/self-registration/adding-sms-templates/add-accountconfirmation-collection.png)
+5. Click on **Add Resource**.
+6. Select **method** as `Create Text Content`.
+7. Provide **name** as `en_us`.
+8. Add the SMS notification template body. **NOTE:** Include `{{confirmation-code}}` placeholder 
+in the SMS template body.
+![add-template-resource](../assets/img/learn/self-registration/adding-sms-templates/add-template-resource.png)
+9. Click on en_us resource and add the following properties.
```

Comment:
Click `en_us` and....

Reply:
Fixed: d72b99b72fc947f18610188db171158cd99f8f08